package.json

```diff
@@ -2,26 +2,32 @@
   "name": "pydata_sphinx_theme",
   "repository": "https://github.com/pydata/pydata-sphinx-theme",
   "scripts": {
-    "build": "webpack"
+    "build": "webpack --mode=production",
+    "build-dev": "webpack",
+    "profile": "webpack --profile --json > bundle-stats.json",
+    "view-profile": "webpack-bundle-analyzer bundle-stats.json"
```

Comment:
Added some useful scripts here to check bundle size and artefacts at compile time